--- newlib-4.4.0.20231231/config.sub	2024-01-01 04:00:18.000000000 +1100
+++ newlib-patched/config.sub	2024-11-17 17:34:24.941883168 +1100
@@ -1704,6 +1704,7 @@
 	     | mpw* | magic* | mmixware* | mon960* | lnews* \
 	     | amigaos* | amigados* | msdos* | newsos* | unicos* | aof* \
 	     | aos* | aros* | cloudabi* | sortix* | twizzler* \
+	     | baremetal* \
 	     | nindy* | vxsim* | vxworks* | ebmon* | hms* | mvs* \
 	     | clix* | riscos* | uniplus* | iris* | isc* | rtu* | xenix* \
 	     | mirbsd* | netbsd* | dicos* | openedition* | ose* \
